app/pathogen/arbovirus/dashboard/ArboStudyPopup.tsx

```diff
@@ -26,6 +26,25 @@ function pathogenTag(pathogen: string) {
     )
 }
 
+function pathogenFullString(pathogen: string) {
```

Comment:
> The url to the study with a hyperlink to that url should be replaced with the name of the study which is still hyperlinked to the url of the study.

Commenting on random line for thread but I'd imagine we can use the `source_sheet_name` as the name of the study for the purposes of the modal. Source sheet names look like this: "A cross-sectional study of acute dengue infection in paediatric clinics in Cameroon" or "Serological and spatial analysis of alphavirus and flavivirus prevalence and risk factors in a rural community in western Kenya"

cc @HimanshuRanka @harrietware @gurickkooner who can correct me if I'm wrong here

Reply:
We solved this in the meeting!